appservice/src/intTest/groovy/nu/yona/server/ActivityTest.groovy

```diff
@@ -1450,10 +1450,11 @@ class ActivityTest extends AbstractAppServiceIntegrationTest
 		reportAppActivity(richardIphone, "NU.nl", "W-1 Mon 04:10", "W-1 Mon 04:20") // Partial overlap
 		reportAppActivity(richardIphone, "NU.nl", "W-1 Mon 05:20", "W-1 Mon 05:30") // Full overlap
 		reportAppActivity(richardIphone, "NU.nl", "W-1 Mon 06:15", "W-1 Mon 06:35") // No overlap at all
+		reportAppActivity(richardIphone, "NU.nl", "W-1 Tue 07:53", "W-1 Tue 08:11") // Next day, so different last monitored activity dates for the devices
```

Comment:
I would prefer a separate set of tests for this feature, instead of combining it with already complex existing tests

Reply:
To me, that wouldn't be right. This test verifies the behavior of a user working on two devices. That test should have asserted the ``lastMonitoredActivityDate`` from the first version, but we forgot to do so. Now I'm correcting the test and the code.